pkg/clients/aws/route53.go

```diff
@@ -613,7 +613,14 @@ func getSTSCredentials(reqLogger logr.Logger, client *sts.STS, roleArn string, e
 			break
 		}
 		if i == (assumeRolePollingRetries - 1) {
-			return nil, fmt.Errorf("timed out while assuming role %s", roleArn)
+			assumeRoleOutput, err = client.AssumeRole(&assumeRoleInput)
+			if err != nil {
+				reqLogger.Error(err, "failed to assumeRole (attempt %d out of %d): %v", i, assumeRolePollingRetries, err)
+				break
```

Comment:
```suggestion
				continue
```
Sorry, I think this was a mistake from a suggestion I made prior